src/test/resources/skat.cov

```diff
@@ -0,0 +1,167 @@
+Sample	Cov1	Cov2	Cov3	Cov4	Cov5	Cov6	Cov7	Cov8	Cov9	Cov10	Cov11	Cov12	Cov13	Cov14	Cov15	Cov16	Cov17	Cov18	Cov19	Cov20	Cov21	Cov22	Cov23	Cov24	Cov25
```

Comment:
I was simply trying to make robust test cases, so far all my test have passed so i'm fine with moving to a smaller test case. These covariates matrices were randomly generated, so I can just randomly generate another one with the desired dimensions.